congress/metadata/L000586.yaml

```diff
@@ -0,0 +1,2 @@
+name: Al Lawson Campaign - Own work
+link: https://upload.wikimedia.org/wikipedia/commons/e/e2/Al_Lawson.jpg
```

Comment:
The page with details is https://en.wikipedia.org/wiki/File:Al_Lawson.jpg, and it says the campaign donated it with a CC-BY-SA 4.0 license. That's better than usual, but we still need to look for a public domain image and not pass attribution requirements on down to downstream users.